integration/conftest.py

```diff
@@ -342,7 +342,7 @@ def cleanup():
     return transport
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture
```

Comment:
Onion domains take a fair bit of time to allocate, don't they? Is there a reason to not cache these at session lifetime?

Reply:
It was causing the tests to fail and I didn't want to spend the time debugging it :cry: